vocabsieve/forvo.py

```diff
@@ -43,35 +47,32 @@ def get_pronunciations(self) -> Forvo:
         else:
             raise Exception("failed to fetch forvo page")
         html = BeautifulSoup(page, "lxml")
-        available_langs_els = html.find_all(
-            id=re.compile(r"language-container-\w{2,4}"))
-        available_langs = [
-            re.findall(
-                r"language-container-(\w{2,4})",
-                el.attrs["id"])[0] for el in available_langs_els]
-        if self.language not in available_langs:
+        available_pronunciations_lists_el = html.find_all(
+            id=re.compile(r"^pronunciations-list-[a-z]+(?:_[a-z]+)?+"))
+        available_lang_locales = [
+            self.get_locale(*re.findall(
+                r"^pronunciations-list-([a-z]+)(?:_([a-z]+))?",
+                el.attrs["id"])[0]) for el in available_pronunciations_lists_el]
+
+        desired_locale = self.get_locale(self.language, self.accent)
+        if not any(desired_locale in s for s in available_lang_locales):
             return self
 
-        lang_container = [
-            lang for lang in available_langs_els if re.findall(
-                r"language-container-(\w{2,4})",
-                lang.attrs["id"])[0] == self.language][0]
-        pronunciations_els = lang_container.find_all(class_="pronunciations")
-        pronunciation_items = pronunciations_els[0].find_all(
-            class_="pronunciations-list")[0].find_all("li")
-        
-        word = self.url.rsplit('/', 2)[-1]
-        headword_el = pronunciations_els[0].find_all('em')[0]
-        headword = headword_el.find_all(text=True)[0].text
-        headword = " ".join(headword.split()[:-2])
+        lang_containers = [
```

Comment:
this doesn't just match the first one anymore because it can match multiple locales. But all of them will append to `pronunciation_items` in the next line